dln/operator.py

```diff
@@ -233,25 +233,25 @@ def _get_completion_response(
         """
         logging.debug(kwargs)
         try:
-            response = openai.Completion.create(
-                engine=self.engine,
-                prompt=prompt_batch,
-                logprobs=top_logprobs or 1,
-                **kwargs,
-            )
-        except openai.InvalidRequestError as e:
+            response = self.client.completions.create(model=self.engine,
+            prompt=prompt_batch,
+            logprobs=top_logprobs or 1,
+            **kwargs)
+        except openai.APIError as e:
             # Retry one by one to find out which prompt is causing the error for debugging
             try:
                 for prompt in prompt_batch:
-                    _ = openai.Completion.create(
-                        engine=self.engine,
-                        prompt=prompt,
-                        logprobs=top_logprobs or 1,
-                        **kwargs,
-                    )
-            except openai.InvalidRequestError as err:
-                self._log_invalid_request_error_message(err, prompt)
-            raise e
+                    _ = self.client.completions.create(model=self.engine,
+                    prompt=prompt,
+                    logprobs=top_logprobs or 1,
+                    **kwargs)
+            except openai.APIError as err:
+                if err.type == 'invalid_request_error':
```

Comment:
Is that documented somewhere? I think I couldn't find it before.

Reply:
I have verified that it is indeed the error type receivd from api after sending an invalid request.